src/components/common/Extensions/UserHub/partials/TransactionsTab/partials/GroupedTransaction.tsx

```diff
@@ -0,0 +1,114 @@
+import React, { FC } from 'react';
+import { FormattedMessage } from 'react-intl';
+
+import { TransactionType } from '~redux/immutable';
+import { arrayToObject } from '~utils/arrays';
+import { formatText } from '~utils/intl';
+import styles from '~frame/GasStation/TransactionCard/GroupedTransaction.css';
+import {
+  getGroupKey,
+  getGroupStatus,
+  getGroupValues,
+} from '~frame/GasStation/transactionGroup';
+import TransactionStatus from './TransactionStatus';
+import GroupedTransactionContent from './GroupedTransactionContent';
+import { GroupedTransactionProps } from '../types';
+
+const displayName =
+  'common.Extensions.UserHub.partials.TransactionsTab.partials.GroupedTransaction';
+
+const GroupedTransaction: FC<GroupedTransactionProps> = ({
+  // appearance,
+  selectedTransactionIdx,
+  transactionGroup,
+  selectedTransaction,
+  unselectTransactionGroup,
+}) => {
+  // const { interactive } = appearance;
+  // const [isOpen, setIsOpen] = useState<boolean>(true);
+
+  const groupKey = getGroupKey(transactionGroup);
+  const status = getGroupStatus(transactionGroup);
+  const values = getGroupValues<TransactionType>(transactionGroup);
+
+  const defaultTransactionGroupMessageDescriptorTitleId = {
+    id: `${
+      transactionGroup[0].metatransaction ? 'meta' : ''
+    }transaction.${groupKey}.title`,
+  };
+  const defaultTransactionGroupMessageDescriptorDescriptionId = {
+    id: process.env.DEBUG
+      ? `${
+          transactionGroup[0].metatransaction ? 'meta' : ''
+        }transaction.debug.description`
+      : `${
+          transactionGroup[0].metatransaction ? 'meta' : ''
+        }transaction.${groupKey}.description`,
+  };
+
+  const value =
+    formatText(
+      {
+        ...defaultTransactionGroupMessageDescriptorTitleId,
+        ...values.group?.title,
+      },
+      values.group?.titleValues || arrayToObject(values.params),
+    ) || '';
+
+  // const handleClose = useCallback(() => {
+  //   //   if (!isDismissible) return;
+  //   setIsOpen(false);
+  //   //   if (typeof callback === 'function') {
+  //   //     callback();
+  //   //   }
+  // }, []);
+
+  // if (!isOpen) return null;
+
+  return (
+    <li>
+      <button
+        type="button"
+        className="w-full"
+        onClick={(event) => unselectTransactionGroup(event)}
```

Comment:
please add an aria-label to the button, use this convention:
"ariaLabel.nameOfAction":